src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -37,10 +37,10 @@ import scala.collection.immutable.ListMap
 
 @clp(group=ClpGroups.VcfOrBcf, description=
   """
-    |Applies one or more filters to a VCF of somatic variants. The VCF must contain genotype information
-    |for the tumor sample.  If the VCF also contains genotypes for one or more other samples, the
-    |`--sample` option must be provided to specify the sample whose genotypes to examine and whose reads
-    |are present in the BAM file.
+    |Applies one or more filters to a VCF of somatic variants.
+    |
+    |If the VCF also contains genotypes for more than one sample, then the `--sample` option must be provided to specify
+    |which genotypes to examine as they relate to the reads in the BAM file (`--bam`).
```

Comment:
```suggestion
    |Applies one or more filters to a VCF of somatic variants. The VCF must contain genotype information
    |for the tumor sample.  If the VCF also contains genotypes for one or more other samples, the
    |`--sample` option must be provided to specify the sample whose genotypes to examine and whose reads
    |are present in the BAM file.```

Reply:
Reverting the leading paragraph because:

a) It's an unrelated change
b) There's no need to break it into multiple paragraphs - I think it actually will lead to people not reading the second couple of sentences
c) The rewording implies you can run this on a VCF without genotypes when that is incorrect

Reply:
My effort was to make it clearer you don't need a tumor sample before running this tool.

There are many somatic samples out there that aren't tumor.

I'll revert... it is an unrelated change!